Update : adding documentation for Kubeflow Pipelines (#3)
inarix/infra-test-devops-google: docs/pipelines/README.md

# Pipelines

> Every explanation means that you have already installed the ```kfctl``` and ```gcloud``` tools before. 

Reviewer: ```suggestion
Considering you already installed the ```kfctl``` and ```gcloud``` tools before. 
```

> Also you will need to have fetch the [kubeflow initial configuration file](https://www.kubeflow.org/docs/started/k8s/kfctl-istio-dex/) and ```kfctl build -f``` this same file to be able to modify some files in the ```.cache``` directory.

Reviewer: ```suggestion
Also you will need to get the [kubeflow initial configuration file](https://www.kubeflow.org/docs/started/k8s/kfctl-istio-dex/) and ```kfctl build -f``` this file to be able to modify the ```.cache``` directory.
```
